locales/de/subtitles/giacomina-castagnetti/giacomina-castagnetti-01.yml

```diff
@@ -106,71 +106,70 @@ de:
           s: 19
           from: 00:01:52,27
           to: 00:01:56,22
-          text: I started going to school in Roncolo.
+          text: Ich ging zur Schule in Roncolo.
         sequence-20:
           s: 20
           from: 00:01:56,22
           to: 00:01:58,22
-          text: ''
+          text: Da sich die Schule in einer landwirtschaftlichen Gegend befand
         sequence-21:
           s: 21
           from: 00:01:58,22
           to: 00:02:08,16
-          text: It only offered courses up to the third grade.
+          text: ging sie nur bis zur dritten Klasse.
         sequence-22:
           s: 22
           from: 00:02:08,16
           to: 00:02:21,23
-          text: I began to attend school with clogs on my feet, a cloth sack, two
-            notebooks -
+          text: Ich ging auf Holzschuhen zur Schule, mit einem Beutel, zwei
+            Notizbüchern -
         sequence-23:
           s: 23
           from: 00:02:21,23
           to: 00:02:30,27
-          text: one ruled and one squared – and a spelling-book
+          text: das eine liniert, das andere kariert – und einer Fibel
         sequence-24:
           s: 24
           from: 00:02:30,27
           to: 00:02:41,11
-          text: which had been passed on by all my brothers before I got it.
+          text: die von all meinen Brüdern weitergereicht wurde bevor ich sie kriegte.
         sequence-25:
           s: 25
           from: 00:02:41,11
           to: 00:02:51,17
-          text: 'At school there was a large banner that had been badly painted, that
-            said:'
+          text: 'In der Schule gab es ein großes Banner das schlecht gemalt war und sagte:'
         sequence-26:
           s: 26
           from: 00:02:51,17
           to: 00:02:54,17
-          text: "“The Duce is always right”."
+          text: "“Der Duce hat immer recht”."
         sequence-27:
           s: 27
           from: 00:02:54,17
           to: 00:03:04,01
-          text: 'Everything was made so that people believed in these things:'
+          text: 'Alles war so organisiert damit Leute an diese Sachen glaubten:'
         sequence-28:
           s: 28
           from: 00:03:04,01
           to: 00:03:13,06
-          text: that the Duce was not to be opposed and that he was always right in
-            what he said.
+          text: dass dem Duce nichts entgegenzusetzen ist und dass er recht hatte in
+            dem was er sagte.
```

Comment:
dass dem Duce nichts entgegenzusetzen sei und er Recht habe in dem, was er sage.